content/applications/inventory_and_mrp/manufacturing/management/subcontracting.rst

```diff
@@ -120,113 +113,103 @@ proposed price coming from the finished products C.
 
-Resupplying subcontractors manually is the simplest approach. To do so,
-create delivery orders in which the subcontractor is set as a delivery
-address and fill the components to deliver.
+Automate Replenishment of Subcontractors
+========================================
 
-To automate the subcontractors’ replenishment propositions, there are
-two approaches:
+There are two ways to automate the supply of raw materials to your 
+subcontractors when purchasing the final product. The chosen method
+depends on wether or not you want the materials to transit through your
+warehouse. Both of these methods are descibred as pull style mechanisms as 
+their trigger is the inital PO to the subcontractor, which creates a need 
+at the subcontracting location, for raw material.  
 
-- Reordering rules;
-- Replenish on order flow.
+If you are supplying your subcontractor with raw material from your own warehouse,
+you must activate the 'Resupply Subcontractor on Order' route as shown 
+below. If this is a component that you buy from a vendor, the buy route
+should also be activated. 
 
-For the first one, just define a reordering rule on the subcontracting
-location, with a minimum and maximum inventory level. When the
-reordering rule is triggered, a delivery order is created to ship the
-components to the subcontractor.
 
-.. image:: media/subcontracting_07.png
+.. image:: media/SBC_6.png
     :align: center
     :alt: 
 
-The second approach is to use a “pull” flow. Here, the demand in the
-finished product (real demand or forecasted one through the Master
-Production Schedule) triggers the replenishment of the subcontractor. To
-do so, select the route *Resupply Subcontractor on Order* on the
-wanted components.
 
-.. image:: media/subcontracting_08.png
+Now, if you want your vendor to resupply your subcontractor directly, you 
+must choose the 'Dropship Subcontractor on Order' option instead. In order 
+for this option to be active on the product form, you must first activate the 
+dropship option from :menuselection:`Purchase --> Configuration -->
+Dropship`.
+Once the PO to the subcontractor is validated, this route will create a dropship 
+RFQ from your vendor to that subcontractor. You then just need to review and validate it. 
+
+
+.. image:: media/SBC_7.png
     :align: center
     :alt: 
 
-Replenishment from another supplier
------------------------------------
+Note that the buy route is not selected in this case, as the dropship route is 
+a buy route already. 
 
-When purchasing items to another supplier, it is possible to ask him to
-deliver the subcontractor directly. To do so, activate the *Drop
-Shipping* feature in :menuselection:`Purchase --> Configuration -->
-Configuration`.
+Finally, if you want to track the stock of these raw materials at your subcontracting
+location(s), then you must activate *Multi-locations* in :menuselection:`Inventory -->
+Configuration --> Storage locations`. 
```

Comment:
Same thing, I would add in Settings after Configuration

Reply:
good point